infrastructure/kctf-samples/apache-php/k8s/deployment/containers.yaml

```diff
@@ -2,3 +2,15 @@ apiVersion: "apps/v1"
 kind: "Deployment"
 metadata:
   name: "chal"
+spec:
+  template:
+    spec:
+      containers:
+      - name: challenge
+        volumeMounts:
+        - name: gcstask
+          mountPath: /mnt/disks/gcs-task
+      volumes:
+      - name: gcstask
+        hostPath:
+          path: /mnt/disks/gcs/apache-php
```

Comment:
what happens if the apache-php folder doesn't exist? Where is that created?

Reply:
this gets created by kubernetes afaict